src/styles.ts

```diff
@@ -40,10 +40,12 @@ export const styles = css`
 		padding-top: var(--paper-input-container_-_padding-top, 8px);
 		padding-bottom: var(--paper-input-container_-_padding-bottom, 8px);
 		position: relative;
-
-		font-family: var(--font-family);
+		border: var(--cosmoz-input-border, none);
+		border-radius: var(--cosmoz-input-border-radius, 0px);
+		max-height: var(--cosmoz-input-max-height);
```

Comment:
We can keep this but in general we shouldn't really touch max-height except in extreme cases.

Reply:
Ok, thanks for the heads-up!